src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -614,13 +614,20 @@ public void testSearch() {
                 .value(new MatcherJson<>(mapper, studiesInfos));
 
         webTestClient.get()
-            .uri("/v1/studies/{studyUuid}/search?q={request}", studyUuid, "equipmentType:(LINE)")
+            .uri("/v1/studies/{studyUuid}/search?userInput={request}&fieldSelector=name", studyUuid, "B")
             .header("userId", "userId")
             .exchange()
             .expectStatus().isOk()
             .expectHeader().contentType(MediaType.APPLICATION_JSON)
             .expectBodyList(EquipmentInfos.class)
             .value(new MatcherJson<>(mapper, linesInfos));
+
+        webTestClient.get()
+            .uri("/v1/studies/{studyUuid}/search?userInput={request}&fieldSelector=bogus", studyUuid, "B")
```

Comment:
We need a test for the expected error message, now we have another message
`{"timestamp":"2021-12-20T13:17:57.489+0000","path":"/v1/studies/573f5563-dd34-45f0-b5d1-979d1c19617d/search","status":400,"error":"Bad Request","message":"Type mismatch.","requestId":"9cdc10bc-1"}`

Reply:
done